templates/vue/src/components/TapestryNode.vue

```diff
@@ -259,7 +258,7 @@ export default {
     },
   },
   mounted() {
-    DragSelectModular.updateSelectableNodes()
+    this.$emit("nodemounted")
```

Comment:
```suggestion
    this.$emit("mounted")
```

Reply:
This is also a lifecycle event name though, so I wanted to differentiate it. I could camelCase it I guess?

Reply:
I don't see an issue with following Narendra's suggestion as it's simpler. I don't think it matters that it's also a lifecycle event name, does it?